File: Prototype Pattern/Minimal Example/Python/concrete_prototype_1.py
from prototype import Prototype

class ConcretePrototype1(Prototype):
    def __init__(self, value):
        self._value = value

    def clone(self):
        return ConcretePrototype1(self._value)
    
    def __str__(self):
        return "value = '" + self._value + "'"

File: Prototype Pattern/Minimal Example/Java/prototypepattern-app/src/main/java/com/dosomedev/ConcretePrototype1.java
package com.dosomedev;

public class ConcretePrototype1 implements Prototype {
    private String value;

    public ConcretePrototype1(String value) {
        this.value = value;
    }

    @Override
    public Prototype clone() {
        return new ConcretePrototype1(this.value);
    }

    @Override
    public String toString() {
        return "value = '" + this.value + "'";
    }
}
